python/ray/scripts/scripts.py

```diff
@@ -208,6 +208,12 @@ def cli(logging_level, logging_format):
     default=None,
     type=str,
     help="Overwrite the options to start Java workers.")
+@click.option(
+    "--java-num-workers-per-process",
+    required=False,
+    default=None,
+    type=int,
+    help="The number of workers per Java worker process.")
```

Comment:
I think it's not necessary to add this option to `ray start`. `ray start` is already very complex.  Then if one wants to config this option, they can use `--internal-config` for now. In the future, we should make C++ config easier to set.

Reply:
The help text of `--internal-config` says that it's only for debugging and development purposes.

Reply:
Done.